packages/visual-editor/src/components/header/Header.tsx

```diff
@@ -147,7 +149,19 @@ const HeaderLayout = (props: HeaderLayoutProps) => {
             </EntityField>
             <button
               className="flex md:hidden ml-auto my-auto"
-              onClick={() => setMenuOpen(!menuOpen)}
+              onClick={() => {
+                analytics?.track({
+                  action: menuOpen ? "COLLAPSE" : "EXPAND",
+                  eventName: "toggleMobileMenu",
+                });
+                setMenuOpen(!menuOpen);
+              }}
+              data-ya-action={
```

Comment:
Too bad we don't do this automatically so someone would need to know to do this. Nbd though.